File: pyprojects/sphinx-jsx/src/py/sphinx_jsx/symbols.py
from pydantic import BaseModel


class JSXSymbols(BaseModel):
    jsx: str = "_jsx"
    jsxs: str = "_jsxs"
    fragment: str = "_Fragment"
    trans: str = "_Trans"
    gettext: str = "_gettext"
    url: str = "_url"

File: pyprojects/sphinx-jsx/src/rs/bundler/symbols.rs
use swc_ecma_transform_i18n::I18nSymbols;
use swc_ecma_utils2::jsx::JSXRuntime;

#[derive(Default)]
pub struct Symbols;

impl JSXRuntime for Symbols {
  const JSX: &'static str = "_jsx";
  const JSXS: &'static str = "_jsxs";
  const FRAGMENT: &'static str = "_Fragment";
}

impl I18nSymbols for Symbols {
  const GETTEXT: &'static str = "_";
  const TRANS: &'static str = "Trans";
}
